sql/core/src/test/resources/mockito-extensions/org.mockito.plugins.MockMaker

```diff
@@ -0,0 +1 @@
+mock-maker-inline
```

Comment:
https://www.baeldung.com/mockito-final#configure-mocktio
Configure Mockito for Final Methods and Classes
<img width="602" alt="image" src="https://user-images.githubusercontent.com/15246973/197348725-9a192ef9-b002-4d1b-be1f-3cff13c937f4.png">